### ISSUE TITLE
Add newpasshash to create-user-tools.sh script

### ISSUE THREAD

**Issue by stvoutsin** [wfau/aglais#793]
Creating new "live" users requires getting a shiro jdbc hash and storing it in our data server. In the most recent deploy, I copied a function from the notes:

```
    newpasshash()
        {
        local password="${1:?}"
        java -jar "${HOME}/lib/shiro-tools-hasher.jar" -i 500000 -f shiro1 -a SHA-256 -gss 128 '${password:?}'
        }
```

It would be useful to add a function like this to the create-user-tools.sh script. Also we need to validate whether the above works or not. In my recent deploy, this was creating a hash for the literal string: "'${password:?}'" rather than replacing. This may be a mistake from my end, as the notes imply that this should work, but it would be good to double check.

**Comment by Zarquan** [comment]
Needs double quotes around the variable to replace in bash.
```
"${password:?}" 
```

**Comment by Zarquan** [comment]
Fixed by #846
